taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+from django.contrib.auth.forms import UserCreationForm
+
+
+def update_license_number(license_number):
+    if len(license_number) != 8:
+        raise ValidationError("License number must be 8 characters long")
+    if (
+        license_number[:3].isalpha() is False
+        or license_number[:3].upper() != license_number[:3]
+    ):
+        raise ValidationError(
+            "License number must start with 3 uppercase letters"
+        )
+    if license_number[3:].isdigit() is False:
+        raise ValidationError("License number must end with 5 digits")
+    return license_number
```

Comment:
This method should be a part of `DriverLicenseUpdateForm`.

Read about `clean_<field>`, so your method name should be `clean_license_number` to validate this field